docs/index.rst

```diff
@@ -19,3 +19,10 @@ The Software Factory documentation is the central space where we collect all use
     swf-blueprint/docs/articles/templates/index
     swf-blueprint/docs/articles/licensing/index
     swf-blueprint/docs/articles/intro/index
+
+.. toctree::
```

Comment:
Interesting use because it nicelly adds a subheader in the sidebar too, I like it.